### ISSUE TITLE
Fix typo on Accessibility page

### ISSUE THREAD

**Issue by amandadupell** [grommet/hpe-design-system#1845]
### Is this issue related to the design system core library or design system website?

Design system site

### Problem

Typo in Accessibility in Design section of the Accessibility page; should be "As an example" instead of just "As example"

### Screenshot of issue 

<img width="500" alt="issue example" src="https://user-images.githubusercontent.com/38971752/132884444-9880c571-3798-47f0-a5a6-05179a53b854.png">
